analytics-provider/src/main/java/org/digma/intellij/plugin/analytics/AnalyticsProvider.java

```diff
@@ -89,4 +90,7 @@ public interface AnalyticsProvider extends Closeable {
     PerformanceMetricsResponse getPerformanceMetrics();
 
     DeleteEnvironmentResponse deleteEnvironment(DeleteEnvironmentRequest deleteEnvironmentRequest);
+
+    // queryParams: (limit, 5), (pageSize, 5) etc
+    String getDashboard(Map<String,String> queryParams);
```

Comment:
is there a reason the signature is not strict one, with field names?

Reply:
its generic, depends on dashboard